src/main/java/games/catan/CatanGame.java

```diff
@@ -28,7 +28,7 @@
 
```

Comment:
Can we also remove the [x]Game classes for each game as part of this update? I feel like those confuse more too.

Reply:
Yes...but I thought you liked them (for testing purposes without getting in the way of Game)

Reply:
I don't :) - I think the easiest is to just use Game. With the exception of the games that do some extra stuff in their Game class, I think Pandemic had some custom competition setup in there.